data/wanderer/wanderers middle.txt

```diff
@@ -8,6 +8,18 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
+mission "Wanderers: Pre-Colony"
+	landing
+	source "Spera Anatrusk"
+	to offer
+		has "event: wanderers: the eye opens"
+		not "event: wanderers: spera anatrusk colony"
+	on offer
+		conversation
+			`With the Eye fully opened, Wanderer ships have begun flooding into this region of space. This desolate planet seems to be of particular interest to them, with drones rapidly surveying in all directions, and a small outpost beginning to form.`
+			`	As you land on the outskirts of this settlement, one of the workers comes up to your ship and explains that this planet will be the staging ground for the Wanderers to explore this new region, but it will take a few weeks to finish. They suggest getting your bearings in these systems before returning to help, as many ships will be needed to help heal the many ruined planets here.`
+				decline
+
```

Comment:
I really like the idea of a pointer, because I am a busy person who occasionally leaves a game for a short while and comes back with no recollection of what my next immediate plan was.I have also attempted to convert this into direct speech for you.

```suggestion
	source "Spera Anatrusk"
	to offer
		has "event: wanderers: the eye opens"
		not "event: wanderers: spera anatrusk colony"
	to complete
		has "event: wanderers: spera anatrusk colony"
	on offer
		conversation
			`With the Eye fully opened, Wanderer ships have begun flooding into this region of space. This desolate planet seems to be of particular interest to them, with drones rapidly surveying in all directions, and a small outpost beginning to form.`
			`	As you land on the outskirts of this settlement, one of the workers comes up to you as you leave your ship. "[Greetings, welcome], it is [delightful, pleasing] to see you have come to show your interest in our exploration. Unfortunately, there exists much [hazard, risk] on a world we know so little about. This will be our staging ground to [explore, catalogue] this new region, but it will take some time to be assured of the [safety, security] of our outpost. I suggest you take time to [orient, locate] yourself in these systems while we set up before returning to help. Of course you are welcome when we have [determined, established] safety, there will be many ships needed to help heal the many ruined planets here.`
				accept
	on visit
		conversation
			`As you land on the outskirts of this settlement, one of the workers comes up to you as you leave your ship. "[Greetings, welcome], it is [delightful, pleasing] to see you but we have not yet finished setting up and ensuring the [safety, security] of our outpost. Please occupy yourself safely for a while until we are [finished, prepared].`

```

Reply:
It may also be confusing to have a mission on your list that does not complete when you visit the marked system, and there's nothing you can do to complete it.

There are plenty of places in the story where you may save between missions.  Perhaps, like with the FW breaks, the Wanderers should try to contact the player when they are ready?

Reply:
I'm in favour of the "player receives a message" dealio for human space stuff, but there'd be no way for them to send you a message except by physically sending a ship after you with a jump drive stolen from the Unfettered. I think by the time players are getting to the Wanderers we might just need to have the mission description say something like "come back after a break" and trust that players know how to read.

Reply:
Hm... well, ignoring my morals for science for a moment, I've played a *bit* of the campaign for research.  I feel dirty, but it strikes me that instead of using a marker we could use the same method to hook the player back into the story as is done when the Wanderers start giving the Hai tribute - the full full text of which is

```
mission "Wanderers: Truce Check"
	landing
	name "Check with Iktat Rek?"
	description "Meet up with Iktat Rek on <planet> to see if there is anything more you can do to help the Wanderers now that they have established a ceasefire with the Unfettered."
	to offer
		has "main plot completed"
		has "event: wanderers: truce timer"
		not "Wanderers: Jump Drive Source: offered"
	source
		government "Wanderer"
		near "Ka'ch'chrai" 1 100
	destination "Vara K'chrai"
	on offer
		conversation
			`It has been a while since you negotiated the ceasefire between the Wanderers and the Unfettered, and so far the truce seems to be holding. Now that the immediate crisis is under control, maybe it's time to meet up with Iktat Rek to talk about finding a more permanent response to the Unfettered incursion.`
				accept```

Reply:
Should that checkup mission only trigger if the player first triggered this new breadcrumb mission, telling them to go check on the colony? Or should it trigger no matter what, where the player receives a message asking them to look at that colony? That's more comprehensive, but would be a bit redundant with the first mission, and also uh oh, more dialog.

Reply:
I would happily swap out the pointer for a reminder of that sort, but I think only if they've triggered the breadcrumb mission. I've attached the suggestion with only the direct dialogue swap out from above for convenience.
```suggestion
	source "Spera Anatrusk"
	to offer
		has "event: wanderers: the eye opens"
		not "event: wanderers: spera anatrusk colony"
	on offer
		conversation
			`With the Eye fully opened, Wanderer ships have begun flooding into this region of space. This desolate planet seems to be of particular interest to them, with drones rapidly surveying in all directions, and a small outpost beginning to form.`
			`	As you land on the outskirts of this settlement, one of the workers comes up to you as you leave your ship. "[Greetings, welcome], it is [delightful, pleasing] to see you have come to show your interest in our exploration. Unfortunately, there exists much [hazard, risk] on a world we know so little about. This will be our staging ground to [explore, catalogue] this new region, but it will take some time to be assured of the [safety, security] of our outpost. I suggest you take time to [orient, locate] yourself in these systems while we set up before returning to help. Of course you are welcome when we have [determined, established] safety, there will be many ships needed to help heal the many ruined planets here.`
				decline

```